plugin/src/main/java/net/aufdemrand/denizen/objects/dPlayer.java

```diff
@@ -1567,6 +1564,35 @@ else if (isOnline()) {
                     .getAttribute(attribute.fulfill(1));
         }
 
+        if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_12_R1)
+                && getPlayerEntity().getOpenInventory().getTopInventory() instanceof MerchantInventory) {

```

Comment:
These checks and object grabs go _inside_ the tags, not outside

Reply:
Will fix.